webpack.common.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
         'shariff/dist/shariff.min.css',
         'select2/dist/css/select2.min.css',
         'slick-carousel/slick/slick.css',
-        'swiper/swiper-bundle.min.css',
```

Comment:
the minified version seems to be no longer exported but since we minify it ourselves that's not a problem